components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/validators/JDBCScopeValidator.java

```diff
@@ -397,8 +397,27 @@ private boolean isUserAuthorizedForScope(String scopeName, String[] userRoles, i
             return false;
         }
 
+	
```

Comment:
We can get rid of unnecessary new lines. WDYT?

Reply:
Added the changes